clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntityFactory.java

```diff
@@ -90,7 +92,7 @@ public void abandonLeadership(String entityIdentifier) {
    * @throws TimeoutException if the creation and configuration of the {@code EhcacheActiveEntity} exceed the
    *        lifecycle operation timeout
    */
-  public void create(final String identifier, final ServerSideConfiguration config)
+  public void create(final String identifier, final ServerSideConfiguration config, final UUID clientId)
```

Comment:
I know we had a chat about this, but now that I see it, I am bothered by having the clientId be injected here instead of generated here. And not having duplications handled as well here.
I believe that either the entity.configure or entity.validate message are a great place to do this validation and thus it could be handled here.
My concern is that with a higher level handling, we kind of tie this id to an object higher in the responsibility chain and may thus have to rework it the moment we need multiple ids for multiple entities.
@AbfrmBlr makes sense?